interfaces/cython/cantera/test/utilities.py

```diff
@@ -27,6 +27,15 @@ def wrapper(*args, **kwargs):
 
     return wrapper
 
+def suppress_temperature_derivative_warnings(test):
+    def wrapper(*args, **kwargs):
+        with warnings.catch_warnings():
+            # suppress warning raised for BlowersMasel and TwoTempPlasma derivatives
+            warnings.simplefilter("ignore")
+            test(*args, **kwargs)
```

Comment:
Also rather than suppressing the warnings, why not test for them? pytest has the `pytest.warns(..., match="")` context manager, which could probably be put into this decorator.